internal/flink/command_region_use.go

```diff
@@ -69,6 +69,12 @@ func (c *command) regionUse(cmd *cobra.Command, _ []string) error {
 		)
 	}
 
+	// Unset the current Flink endpoint if there is a cloud/region change based on discussion with Flink product team
+	output.Println(c.Config.EnableColor, "Reset the current Flink endpoint to empty due to the cloud or region switch")
```

Comment:
```suggestion
	output.Println(c.Config.EnableColor, "The current Flink endpoint has been unset due to the cloud or region change.")
```
Suggestion: change the wording because the current wording could be misinterpreted as an instruction to the user to run the unset command

Reply:
Absolutely, I read this message again and feel the same way as you pointed out, updated.